site/_data/i18n/authors.yaml

```diff
@@ -1356,3 +1356,8 @@ tristramsouthey:
     en: 'Tristram Southey'
   description:
     en: 'Product Manager, Google Chrome'
+jyasskin:
+  title:
+    en: 'Jeffrey Yasskin'
+  description:
+    en: 'Software engineer working on Web Standards for Chrome.'
```

Comment:
We're just "Web Standards Tech Leads, Chrome Team" at the signature, which is good.  I think here, I'd just leave it as two separate entries (because I think this comes from a template) with "Chrome Web Standards Lead" for both of us.